### ISSUE TITLE
Allow "{" on next line after class definition

### ISSUE THREAD

**Issue by linslin** [PHPCheckstyle/phpcheckstyle#88]
I like to konw how to allow this kind of block opening in class cases:

```
class MyClass
{
}
```

This pattern currently result in this warnings: `checkWhiteSpaceBefore` & `funcDefinitionOpenCurly`.

Thanks in advance!

**Comment by tchule** [comment]
Hi,

You can edit the config file and change the rule from "same line" to "new line" by setting the value to "nl".

```
 <!-- Check the position of the open curly brace after a function -->
	<!-- sl = same line -->
	<!-- nl = new line -->
	<test name="funcDefinitionOpenCurly">
		<property name="position" value="sl" />
	</test>
```
